service-commands/scripts/create-instance.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Usage: create-instance.sh [options] <name> 
+#
+# Options:
+#   -p <provider>
+#   -i <image>
+#   -d <disk-size>
+#   -m <machine-type>
+
+set -e
+
+PROTOCOL_DIR=${PROTOCOL_DIR:-$(dirname $(realpath $0))/../../}
```

Comment:
i saw this locally 
```
➜  audius-protocol git:(vss-A-refactor) ✗ A create-instance dm-discovery-test-2 --provider azure
bash: /usr/local/bin/../..//service-commands/scripts/create-instance.sh: No such file or directory
```

Reply:
i was able to export PROTOCOL_DIR and it worked which i guess is the intent but i actually think it might be easier to force PROTOCOL_DIR to be set